acme-cli/src/main/java/io/micronaut/acme/cli/DeactivateAccountCommand.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.acme.cli;
+
+import org.shredzone.acme4j.Account;
+import org.shredzone.acme4j.AccountBuilder;
+import org.shredzone.acme4j.Login;
+import org.shredzone.acme4j.Session;
+import org.shredzone.acme4j.exception.AcmeException;
+import org.shredzone.acme4j.util.KeyPairUtils;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.security.KeyPair;
+import java.util.concurrent.Callable;
+
+/**
+ * Allows for deactivating of an existing Acme account on the given acme server.
+ */
+@CommandLine.Command(name = "deactivate-account",
+        aliases = {"deactivate", "da"},
+        description = "Deactivates an existing Let's Encrypt account",
+        usageHelpWidth = 95,
+        helpCommand = true
```

Comment:
Shouldn't this be false/the default value?
This appears to be why tests are failing.

The command returns `1` on failure, but it's marked as a help command so it always returns `parsed.getCommandSpec().exitCodeOnUsageHelp();` which happens to be `0`.

picocli/CommandLine.java is too big to link to, but you can see the code here:
<img width="1002" alt="Screen Shot 2020-04-12 at 11 05 50 PM" src="https://user-images.githubusercontent.com/131809/79088943-3f1d7580-7d12-11ea-9091-2183ac1c1060.png">

Reply:
Yup Yup.... i misunderstood ... c51179e